packages/e2e-tests/specs/rtl.test.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	getEditedPostContent,
+} from '@wordpress/e2e-test-utils';
+
+// Avoid using three, as it looks too much like two with some fonts.
+const ARABIC_ZERO = '٠';
+const ARABIC_ONE = '١';
+const ARABIC_TWO = '٢';
+
+describe( 'RTL', () => {
+	beforeEach( async () => {
+		await createNewPost();
+	} );
+
+	it( 'should arrow navigate', async () => {
+		await page.evaluate( () => document.dir = 'rtl' );
```

Comment:
Do you think that's the best way to test Arabic? Do you think it would be good to have a utility in our e2e-test-utils package to change WordPress's language?

I fear this change could be considered an implementation details, say tomorrow we base the behavior on `isRTL` editor setting for instance, we might have to revisit the tests?

Reply:
I tried, but I couldn't get it to work. `document.dir` seems to be enough though.

Also, I don't think `isRTL` would be a good idea. If you want to change the direction of a specific `RichText` instance, you need to set the `dir` attribute on the wrapper. This is the standard. :)